apollo-converters/moshi/build.gradle

```diff
@@ -3,11 +3,18 @@ apply plugin: 'java'
 targetCompatibility = JavaVersion.VERSION_1_7
 sourceCompatibility = JavaVersion.VERSION_1_7
 
+configurations {
```

Comment:
I thought compileOnly is supported by gradle and we don't need custom configuration?